compose.yaml

```diff
@@ -5,15 +5,20 @@ services:
     init: true
     build:
       context: .
+      target: development
+    env_file: .env.docker
     environment:
-      API_URI: ${API_URI}
-      DATABASE_URL: postgres://${PG_USER}:${PG_PASSWORD}@database:5432/${PG_DATABASE}
+      API_URI: ${API_URI-http://localhost:2300}
+      DATABASE_URL: postgres://${PG_USER:-postgres}:${PG_PASSWORD:-password}@database:5432/${PG_DATABASE:-terminus_development}
     ports:
       - "2300:2300"
     restart: unless-stopped
+    volumes:
+      - .:/app
     depends_on:
       database:
         condition: service_healthy
+    command: ["bin/docker/dev"]
```

Comment:
:axe: As mentioned above, this configuration isn't meant for devleopment.

Reply:
Addressing in response to your larger PR comment